.lando/scripts/appserver_build.sh

```diff
@@ -4,59 +4,15 @@
 DRUPAL_ROOT=/app/web
 DRUPAL_CUSTOM_CODE=$DRUPAL_ROOT/modules/custom
 
-# Semaphore files to control whether we need to trigger an install
-# of supporting software or config files.
-NODE_YARN_INSTALLED=/etc/NODE_YARN_INSTALLED
```

Comment:
I suspect much of this is legacy stuff from NIDirect. The variables can be set in other ways (env vars, for instance) but we also don't make use of yarn or phpunit much anymore. Better to add them in when needed than leave it dangling